src/Kernel-Chronology-Extras/Month.class.st

```diff
@@ -45,8 +45,8 @@ Month class >> february [
 
 { #category : #accessing }
 Month class >> indexOfMonth: aMonthName [ 
```

Comment:
Can you provide example calls here (e.g.:  `Month indexOfMonth: 'Jan' >>> 1` or similar )?